Upstream: yes
Source: http://svn.debian.org/wsvn/pkg-shadow/upstream/trunk/src/groupmod.c?op=diff&rev=0&sc=0
Summary: Fix "groupmod: cannot determine your user name" error

diff --git src/groupmod.c src/groupmod.c
index 4205df2..da6d77f 100644
--- src/groupmod.c
+++ src/groupmod.c
@@ -724,7 +724,7 @@ int main (int argc, char **argv)
 	{
 		struct passwd *pampw;
 		pampw = getpwuid (getuid ()); /* local, no need for xgetpwuid */
-		if (NULL == pamh) {
+		if (NULL == pampw) {
 			fprintf (stderr,
 			         _("%s: Cannot determine your user name.\n"),
 			         Prog);
